canonical-webteam/documentation-builder
# Add Snap Enterprise Proxy homepage example

## Done

- Added Snap Enterprise Proxy homepage example to documentation-builder docs

## QA

- Pull code
- Open `docs/metadata.yaml` and replace contents with:
``` yaml
site_title: Snap Enterprise Proxy documentation
```
- Run `./run`
- Go to http://0.0.0.0:8000/en/homepage-examples/enterprise-proxy-example
- Check that it matches the [design](https://raw.githubusercontent.com/ubuntudesign/vanilla-design/master/Templates/Documentation-homepage/005%20Enterprise%20proxy%20template/enterprise-proxy-documentation-home-template.png)

## Screenshot

![0 0 0 0_8000_en_homepage-examples_enterprise-proxy-example](https://user-images.githubusercontent.com/25733845/41469388-d8a207e2-70a4-11e8-8ada-5746ffc5352d.png)
